src/main/java/xyz/nucleoid/plasmid/storage/PlayerScoreStorage.java

```diff
@@ -0,0 +1,62 @@
+package xyz.nucleoid.plasmid.storage;
+
+import net.fabricmc.fabric.api.util.NbtType;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.ListTag;
+import net.minecraft.nbt.Tag;
+import net.minecraft.server.network.ServerPlayerEntity;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ * A simple class for storing the scores of players.
+ * 'Score' can mean any number.
+ */
+public class PlayerScoreStorage implements ServerStorage {
+    public final Map<UUID, Float> scoreMap = new HashMap<>();
```

Comment:
This can use an `Object2Float` map

Reply:
It might also make sense to use an int just for precision-loss purposes?

Reply:
I made it not use an int since some games may want to store scores that would be decimals, such as how fast you completed a game.
Although yes the precision-loss could be a problem, so I could make it use double instead.

Reply:
Double would be better, but it's still not possible to guarantee total accuracy. I think if something wants to store decimal points they could store with some constant denominator?

Reply:
I suppose someone could just store it with a constant denominator and could always just make their own score storage.
I'll make it use an `Object2IntMap` instead then 👍